src/main/java/com/zufar/onlinestore/cart/api/CartApiImpl.java

```diff
@@ -0,0 +1,81 @@
+package com.zufar.onlinestore.cart.api;
+
+import com.zufar.onlinestore.cart.converter.ShoppingSessionDtoConverter;
+import com.zufar.onlinestore.cart.dto.AddNewItemToShoppingSessionRequest;
+import com.zufar.onlinestore.cart.dto.GetShoppingSessionRequest;
+import com.zufar.onlinestore.cart.dto.RemoveItemFromShoppingSessionRequest;
+import com.zufar.onlinestore.cart.dto.ShoppingSessionDto;
+import com.zufar.onlinestore.cart.dto.UpdateProductsQuantityInShoppingSessionItemRequest;
+import com.zufar.onlinestore.cart.entity.ShoppingSession;
+import com.zufar.onlinestore.cart.entity.ShoppingSessionItem;
+import com.zufar.onlinestore.cart.exception.InvalidShoppingSessionIdInUpdateProductsQuantityRequestException;
+import com.zufar.onlinestore.cart.exception.ShoppingSessionItemNotFoundException;
+import com.zufar.onlinestore.cart.exception.ShoppingSessionNotFoundException;
+import com.zufar.onlinestore.cart.repository.ShoppingSessionItemRepository;
+import com.zufar.onlinestore.cart.repository.ShoppingSessionRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Isolation;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.Optional;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class CartApiImpl implements CartApi {
+
+    private final ShoppingSessionRepository shoppingSessionRepository;
+    private final ShoppingSessionItemRepository shoppingSessionItemRepository;
+    private final ShoppingSessionDtoConverter shoppingSessionDtoConverter;
+
+    @Override
+    public ShoppingSessionDto getShoppingSession(final GetShoppingSessionRequest getShoppingSessionRequest) {
+        return null;
```

Comment:
Is it possible?

Reply:
yes, it will be implemented in another PR